### PR TITLE
Update evaluation and prediction docs.

### PR DESCRIPTION
**Pull request by KazusaW1999**
1. Split evaluation doc into two docs. Prediction doc is independent now.
2. Add custom color map instruction.
3. Modify href.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Welcome to PaddleSeg! PaddleSeg is an end-to-end image segmentation development
     * [Create Your Own Model](./docs/design/create/add_new_model.md)
 * [Model Training](/docs/train/train.md)
 * [Model Evaluation](./docs/evaluation/evaluate/evaluate.md)
+* [Predicting and Visualization](./docs/predict/predict.md)
```
Comment (michaelowenliu) [inline README.md]: Prediction

File: docs/evaluation/evaluate/evaluate.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+## Model Evaluating
 
-### 1.**配置化驱动**方式下评估和预测
+### 1. Evaluation and Prediction under **Configuration-Driven** Approach
 
-#### 评估
+#### Evaluating
 
-训练完成后，用户可以使用评估脚本val.py来评估模型效果。假设训练过程中迭代次数（iters）为1000，保存模型的间隔为500，即每迭代1000次数据集保存2次训练模型。因此一共会产生2个定期保存的模型，加上保存的最佳模型best_model，一共有3个模型，可以通过model_path指定期望评估的模型文件。
+After the training is completed, the user can use the evaluation script val.py to evaluate the effect of the model. Assuming that the number of iterations (iters) in the training process is 1000, the interval for saving the model is 500, that is, the training model is saved twice for every 1000 iterations of the data set. Therefore, a total of 2 regularly saved models will be generated, plus the best saved model `best_model`, there are a total of 3 models, and the model file that you want to evaluate can be specified by `model_path`.
```
Comment (michaelowenliu) [inline docs/evaluation/evaluate/evaluate.md]: After the training, the user ...

File: README.md (hunk 1 above)
Comment (KazusaW1999) [reply README.md]: Done.

File: docs/evaluation/evaluate/evaluate.md (hunk 2 above)
Comment (KazusaW1999) [reply docs/evaluation/evaluate/evaluate.md]: Done.

**Review by michaelowenliu — APPROVED** [review]

**Pull request merged**